home/modules/ROOT/pages/index.adoc

```diff
@@ -8,10 +8,17 @@
 We're no longer updating the content on this site regularly. Check out the https://www.couchbase.com/support-policy[Couchbase Support Policy] for information on version support and End of Life schedule.
 
 ifdef::primary-site-url[]
-Click {primary-site-url}[here] to return to the main documentation site.
+This page contains links to End-of-Life, archived documentation, for reference purposes, as well as links to docs for current, GA products. Click {primary-site-url}[here] to return to the main documentation site.
 endif::[]
 --
 
+
+[.tiles.browse]
+== Couchbase Capella
+
+Capella is an -as-a-service offering of our operational and analytics databases, App Services to connect edge devices, and AI Services (in Developer Preview).
```

Comment:
Do we use "-as-a-service" like that?

Reply:
I can't remember whether I got it from the .com site or a PM blog.
But yes, it's not unambiguously great, is it?  :thinking: